packages/core/test-utils/package.json

```diff
@@ -17,13 +17,21 @@
     "@parcel/core": "2.7.0",
     "@parcel/fs": "2.7.0",
     "@parcel/package-manager": "2.7.0",
+    "@parcel/types": "2.7.0",
     "@parcel/utils": "2.7.0",
     "chalk": "^4.1.0",
     "ncp": "^2.0.0",
     "nullthrows": "^1.1.1",
     "posthtml": "^0.16.5",
     "posthtml-parser": "^0.10.1",
-    "resolve": "^1.12.0",
+    "resolve": "^1.22.1",
```

Comment:
Updated this to the lasted minor version as there were issues with `linux-arm64`